src/client/dfs/SConscript

```diff
@@ -1,17 +1,15 @@
 """Build FUSE client"""
 import daos_build
 
-def scons():
-    """Execute build"""
-    Import('env', 'prereqs')
-
-    denv = env.Clone()
-    denv.AppendIfSupported(CCFLAGS='-fvisibility=hidden')
-
-    """ If Lustre installed build a Lustre-aware libduns """
+def configure_lustre(denv):
+    """Do Lustre configure checks"""
+    if GetOption('help'):
```

Comment:
This makes me think that this could also be avoided in case of clean/-c , right ?

Reply:
possibly, yes.